core/src/test/java/org/apache/iceberg/rest/TestRESTScanPlanning.java

```diff
@@ -424,7 +439,11 @@ public void metadataTablesWithRemotePlanning(MetadataTableType type) throws IOEx
     // tasks, this test just verifies that metadata tables can be scanned with RESTTable.
     Table metadataTableInstance = MetadataTableUtils.createMetadataTableInstance(table, type);
     assertThat(metadataTableInstance).isNotNull();
-    assertThat(metadataTableInstance.newScan().planFiles()).isNotEmpty();
+    if (!type.equals(MetadataTableType.POSITION_DELETES)) {
```

Comment:
nit: you might want to switch the condition to have `if type is position_deletes then do X else do Y`